.github/workflows/ship.yml

```diff
@@ -0,0 +1,26 @@
+name: Ship
+
+on:
+  workflow_dispatch: # manual override
+  schedule:
+    - cron: "0 * * * *" # every hour
+
+jobs:
+  ship:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Deno repo
+        uses: actions/checkout@v2
+        with:
+          repository: "denoland/deno"
+          fetch-depth: 1
+          submodules: false
+
+      - name: Setup Deno
+        uses: denolib/setup-deno@v2
+
+      - name: Check and publish
+        run: |
+          deno run -A --unstable https://denopkg.com/nestdotland/eggs/mod.ts link --key ${{ secrets.NESTAPIKEY }}
```

Comment:
Is there a reason we're using denopkg.com in this first `deno run` instead of `x.nest.land`? I'd assume getting the latest version from master?

Reply:
yes that is the reason. we are using the latest deno so we need latest eggs too. or we need to lock both of them.

should i lock it?

Reply:
No, you raise a good point. This is actually a good example of a time when a latest version is needed from our registry... Interesting.